ChallengePart1.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Chick Weight Analysis"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(knitr)
+```
+
+```{r readin,echo=FALSE}
+chicks <- read.csv('chickwts.txt', header = TRUE, sep = ",")
+```
+
+We have data which details the **weight** of different chicken bracketted by the type of **feed* they ate.  The whole of the data is summarized below:
+
+```{r summary, echo=FALSE}
+summary(chicks)
+```
+
+We summarize the data for **feed type** cassien below:
+
+```{r cassein summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="cassein"),])
+```
+
+We summarize the data for **feed type** horsebean below:
+
+```{r horsebean summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="horsebean"),])
+```
+
+We summarize the data for **feed type** linseed below:
+
+```{r linseed summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="linseed"),])
+```
+
+We summarize the data for **feed type** meatmeal below:
+
+```{r meatmeal summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="meatmeal"),])
+```
+
+We summarize the data for **feed type** soybean below:
+
+```{r soybean summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="soybean"),])
+```
+
+We summarize the data for **feed type** sunflower below:
+
+```{r sunflower summary, echo=FALSE}
+summary(chicks[which(chicks$feed=="sunflower"),])

```

Comment:
summary is not really necessary. But, the code for the analysis should be provided in the R Markdown document.